### PR TITLE
fix: bunyan logs rotating file are broken, don't use them

### PR DESCRIPTION
**Pull request by Bilb**
changed based to unstable as we don't want this in the release with onboarding

Note: maybe we should just not display the logs and just copy them to desktop

### CODE REVIEW

**Review by yougotwill — CHANGES_REQUESTED** [review]
Tested on my machine and it seems to be working fine. Delete logs folder and it correctly remakes it.
The test will be in the next release that other's logging has been fixed

File: ts/node/logging.ts [hunk 1]
```diff
@@ -105,85 +107,33 @@ async function deleteAllLogs(logPath: string) {
   });
 }
 
-async function cleanupLogs(logPath: string) {
+async function cleanupLogs(logFile: string, logFolder: string) {
   const now = new Date();
   const earliestDate = new Date(
-    Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate() - 6)
+    Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate() - 2) // keeping 2 days worth of logs at most
```
[the remark this answers is not in the document]
Comment (Bilb) [reply ts/node/logging.ts]: updated

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

File: ts/components/AboutView.tsx [hunk 2]
```diff
@@ -54,8 +54,8 @@ export const AboutView = () => {
   }
 
   const versionInfo = `v${window.getVersion()}`;
-  const commitInfo = `Commit ${window.getCommitHash()}` || '';
-  const osInfo = `${window.getOSRelease()}`;
+  const systemInfo = `${window.i18n('systemInformationDesktop', [window.getOSRelease()])}`;
+  const commitInfo = `${window.i18n('commitHashDesktop', [window.getCommitHash() || window.i18n('unknown')])}`;
```
Comment (Bilb) [inline ts/components/AboutView.tsx:58]:
```suggestion
  const systemInfo = window.i18n('systemInformationDesktop', [window.getOSRelease()]);
  const commitInfo = window.i18n('commitHashDesktop', [window.getCommitHash() || window.i18n('unknown')]);
```

File: ts/mains/main_node.ts [hunk 3]
```diff
@@ -648,79 +648,37 @@ async function showAbout() {
...
-        throw Error(`${err}`);
-      }
-      console.info(`Saved log - ${outputPath}`);
-    });
+    const loggerFilePath = getLoggerFilePath();
+    if (!loggerFilePath) {
+      throw Error('No logger file path');
+    }
+
+    fs.copyFileSync(loggerFilePath, outputPath);
```
Comment (Bilb) [inline ts/mains/main_node.ts:674]: have you checked how slow that it for a 20MB file by any chance?

Comment (Bilb) [reply ts/mains/main_node.ts:674]: we might need to use the non sync one

File: ts/mains/main_node.ts [hunk 4]
```diff
@@ -648,79 +648,37 @@ async function showAbout() {
...
+    console.info(`[log] Copied logs to ${outputPath} from ${loggerFilePath}`);
+
+    // append any additional info
+    if (additionalInfo) {
+      fs.appendFileSync(outputPath, additionalInfo, { encoding: 'utf-8' });
```
Comment (Bilb) [inline ts/mains/main_node.ts:679]: same comment

File: ts/mains/main_node.ts (hunk 3 above)
Comment (yougotwill) [reply ts/mains/main_node.ts]: Done!

File: ts/mains/main_node.ts (hunk 4 above)
Comment (yougotwill) [reply ts/mains/main_node.ts]: Fixed

File: ts/components/AboutView.tsx (hunk 2 above)
Comment (yougotwill) [reply ts/components/AboutView.tsx]: Fixed

**Review by yougotwill — COMMENTED** [review]
Updated

**Comment by Bilb** [comment]
LGTM

**Pull request merged**